helpers/ipfs.ts

```diff
@@ -12,17 +13,35 @@ type Data = {
   name: string;
   description: string;
   blockchain: string;
+  type: string;
 };
 
-type UploadData = {
-  image: File;
-  name: string;
-  description: string;
+export const uploadToIPFS = async (data: Blob | File) => {
+  if (!data) throw Error("Data is invalid");
```

Comment:
Is this line needed?? Is there ever an instance that the data will not exist?? Should it not get caught by the yup validation? Also if !data then TS should error I think??